boards/risc-v/mpfs/common/src/mpfs_board_spi.c

```diff
@@ -95,5 +97,28 @@ int mpfs_board_spi_init(void)
 
 #endif /* CONFIG_SPI_DRIVER */
 #endif /* CONFIG_MPFS_SPI1 */
+
+#ifdef CONFIG_MPFS_CORESPI
+  /* REVISIT: Initialize all of them ? */
```

Comment:
add a not-implemented #error here if more than 1 corespi is configured, or add initialization of all the configured ones here?

Reply:
We don't use this init file so I deemed it unnecessary to add that here. I can of course do it...

Reply:
you are right, no need